admin/tests/unit/services/error-notifier-test.js

```diff
@@ -0,0 +1,65 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+
+module('Unit | Service | error-notifier', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    const service = this.owner.lookup('service:error-notifier');
+    assert.ok(service);
+  });
+
+  test('it notifies a non-JSONAPI error as a single notification', function(assert) {
+    // given
+    const service = this.owner.lookup('service:error-notifier');
+    const errorStub = sinon.stub();
```

Comment:
C'est plus un mock si on vérifie qu'il a été appelé ?
```suggestion
    const errorMock = sinon.stub();
```